extensions/wikia/PortableInfobox/tests/nodes/NodeImageTest.php

```diff
@@ -11,12 +11,12 @@ protected function setUp() {
 	 */
 	public function testGalleryData() {
 		$input = '<div data-model="[{&quot;caption&quot;:&quot;_caption_&quot;,&quot;title&quot;:&quot;_title_&quot;}]"></div>';
-		$expected = array(
-			array(
+		$expected = [
+			[
 				'label' => '_caption_',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "label" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)